package.json

```diff
@@ -38,6 +37,7 @@
   },
   "homepage": "https://github.com/craftship/yith#readme",
   "dependencies": {
+    "aws-sdk": "^2.7.10",
```

Comment:
Ah I see. The issue I see with this approach, is that there's not a single source of truth for *all* your dependencies - i.e you're dependent on a globally-defined dependency in the Lambda runtime. You _may_ find that this dependency gets bumped independently of what is (currently) defined in your `devDependencies`, leading to a disparity between local and Lambda environments